server/migrations/20180716204138-create-user.js

```diff
@@ -0,0 +1,65 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER,
+    },
+    username: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+    },
+    firstName: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    lastName: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    googleId: {
+      type: Sequelize.STRING,
+      allowNull: true,
+    },
+    facebookId: {
+      type: Sequelize.STRING,
+      allowNull: true,
+    },
+    twitterId: {
+      type: Sequelize.STRING,
+      allowNull: true,
+    },
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+    },
+    password: {
+      type: Sequelize.STRING,
+      allowNull: true,
+    },
+    verified: {
+      type: Sequelize.BOOLEAN,
+      defaultValue: false,
+    },
+    bio: {
```

Comment:
Please the user's bio datatype should be a `TEXT` and not a `STRING`, this is because a `STRING` is limited to about 255 characters while `TEXT` can store up to 5,000 characters.

Reply:
Thanks for the review, however if you check the code base for the user reset on the branch you it has been updated to TEXT after rebase process. Thanks.